code/modules/mob/living/emote.dm

```diff
@@ -79,6 +79,7 @@
 	message_larva = "lets out a sickly hiss of air and falls limply to the floor..."
 	message_monkey = "lets out a faint chimper as it collapses and stops moving..."
 	message_simple =  "stops moving..."
+	cooldown = (15 SECONDS)

```

Comment:
```suggestion
	cooldown = (60 MINUTES)
```

Reply:
I would actually be lynched out of the community.